### PR TITLE
Add upgrade tech notes

### PR DESCRIPTION
**Pull request by zkayyali812**

### CODE REVIEW

File: stories/topics/proc-aws-running-upgrade.adoc [hunk 1]
```diff
@@ -1,6 +1,12 @@
 [id="proc-aws-running-upgrade"]
 
-= Running the ansible-on-clouds-ops 2.4 container to upgrade your {PlatformNameShort} deployment
+= Running the upgrade playbook
+
+[NOTE]
+=====
+{PlatformNameShort} upgrade to 2.4.20230630 will update the listeners on its internal load balancers. If a resource dependancy was added after installation, this will need to be temporarily alleviated to allow the upgrade to succeed. See xref:tech-note-aws-upgrade[technical note] for more detail.
```
Comment (ianf77) [inline stories/topics/proc-aws-running-upgrade.adoc:7]: {PlatformNameShort} upgrade to 2.4.202306630 updates the listeners on its internal load balancers. 

what does 'temporariy alleviated' mean?

File: stories/topics/proc-gcp-running-upgrade.adoc [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 = Running the upgrade playbook
 
+[NOTE]
+=====
+{PlatformNameShort} upgrade to 2.4.20230630 will update the protocol of its internal load balancers. If additional networking configurations were added after installation, they may also need to be updated to ensure connectivity. See xref:tech-note-gcp-upgrade[technical note] for more detail.
```
Comment (ianf77) [inline stories/topics/proc-gcp-running-upgrade.adoc:7]: {PlatformNameShort} upgrade to 2.4.20230630 updates the protocol of...

they can  also require updating to ensure...

For more information, read xref:tech-note-gcp-upgrade[technical note].

File: stories/topics/tech-note-aws-upgrade.adoc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[id="tech-note-aws-upgrade"]
+
+= Upgrade note
+
+Upgrading {PlatformNameShort} to 2.4.20230630 will update the protocol of the listeners attached to the internal load balancers from HTTP to HTTPs. In order to update the listener protocol in AWS, the listener must be deleted and recreated properly. In order for this operation to succeed there must not have been an added resource dependancy on the listener. If there is a resource dependancy on the listener, this will halt the upgrade, and the dependancy must be manually alleviated for the upgrade to succeed. After the upgrade has suceeded, the dependant resource can be recreated. It is recommended to revalidate additional added networking configurations after upgrade.
```
Comment (ianf77) [inline stories/topics/tech-note-aws-upgrade.adoc:5]: Upgrading {PlatformNameShort} to 2.4.20230630 updates the protocol...

To update the listener protocol....

For this operation to succeed....

on the listener, this halts the upgrade

(manually alleviated again) ?

When the upgrade has succeeded....

You must revalidate any additional networking....

File: stories/topics/tech-note-gcp-upgrade.adoc [hunk 4]
```diff
@@ -0,0 +1,5 @@
+[id="tech-note-gcp-upgrade"]
+
+= Upgrade note
+
+Upgrading {PlatformNameShort} to 2.4.20230630 will update the protocol of its internal load balancers from HTTP to HTTPs. If additional networking configurations were added, they may also need to be updated to ensure connectivity. After the upgrade has suceeded, it is recommended to revalidate additional added networking configurations.
```
Comment (ianf77) [inline stories/topics/tech-note-gcp-upgrade.adoc:5]: Upgrading {PlatformNameShort} to 2.4.20230630 updates the protocol...

they can also be updated....

When the upgrade has succeeded, you must revalidate any...

**Review by ianf77 — COMMENTED** [review]
Some comments

File: stories/topics/proc-aws-running-upgrade.adoc (hunk 1 above)
Comment (zkayyali812) [reply stories/topics/proc-aws-running-upgrade.adoc:7]: I will update this to say "temporarily removed"

File: stories/topics/proc-aws-running-upgrade.adoc [hunk 5]
```diff
@@ -1,6 +1,12 @@
 [id="proc-aws-running-upgrade"]
 
-= Running the ansible-on-clouds-ops 2.4 container to upgrade your {PlatformNameShort} deployment
+= Running the upgrade playbook
+
+[NOTE]
+=====
+{PlatformNameShort} upgrade to 2.4.20230630 will update the listeners on its internal load balancers. If a resource dependancy was added after installation, this will need to be temporarily removed to allow the upgrade to succeed. See xref:tech-note-aws-upgrade[technical note] for more detail.
```
Comment (ianf77) [inline stories/topics/proc-aws-running-upgrade.adoc:7]: {PlatformNameShort} upgrade to 2.4.20230630 updates the listener...

after installation, this must be temporarily removed for the upgrade to succeed.

**Review by ianf77 — COMMENTED** [review]
One comment

**Pull request merged**